normandy/base/templatetags/app_data.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+from django.template import Node
+from django.template import Library
+from django.utils.safestring import mark_safe
+
+from normandy.base.api.serializers import UserSerializer
+
+
+register = Library()
+
+
+@register.tag
+def app_data(parser, token):
+    return AppDataNode()
+
+
+class AppDataNode(Node):
```

Comment:
This should probably just be a [simple tag](https://docs.djangoproject.com/es/1.9/howto/custom-template-tags/#django.template.Library.simple_tag).